Release/include/pplx/pplxtasks.h

```diff
@@ -136,13 +136,17 @@ namespace std
     }
 }
 #endif /* _MSC_VER < 1700 */
-#ifndef _PPLTASK_ASYNC_LOGGING
```

Comment:
It seems like a better change would be to avoid this mess by just renaming this to PPLX_TASK_ASYNC_LOGGING ?